doc/classes/Node3D.xml

```diff
@@ -314,8 +314,8 @@
 			[b]Note:[/b] If the node is not inside the tree, getting this property fails and returns [constant Vector3.ZERO].
 		</member>
 		<member name="global_transform" type="Transform3D" setter="set_global_transform" getter="get_global_transform">
-			The transformation of this node, in global space (relative to the world). Contains and represents this node's [member global_position], [member global_rotation], and global scale.
-			[b]Note:[/b] If the node is not inside the tree, getting this property fails and returns [constant Transform3D.IDENTITY].
+			World space (global) [Transform3D] of this node, i.e. the combined transform up to the topmost [Node3D] node. The topmost item is a [Node3D] that either has no parent, has non-[Node3D] parent or it has [member top_level] enabled.
```

Comment:
For now I just kept my old documentation bit when resolving the conflict, but if this were to be merged it should likely adapted to some of the changes in the new string (like not mentioning [Transform3D] explicitly which is redundant with the returned type). But overall my version provides more useful details IMO.